docs/content/specs-defs/specs/shared/_index.md

```diff
@@ -1013,6 +1013,22 @@ Module owners **MAY** also have to provide additional outputs depending on the I
 
 <br>
 
+#### ID: RMFR8 - Category: Composition - End-of-life resource versions
+
+When a given version of an Azure resource used in a resource module reaches its end-of-life (EOL) and is no longer supported by Microsoft, the module owner **MUST** ensure that:
+
+1. The module is aligned with these changes and only includes supported versions of the resource. (This is typically achieved through the allowed values in the parameter that specifies the resource version.)
+2. The following notice is shown under the `Notes` section of the module's `readme.md`. (If any related public announcement is available, it can also be linked to from the Notes section.):
+    > "Certain versions of this Azure resource reached their end of life. The latest version of this module only includes supported versions of the resource. All unsupported versions have been removed from the related parameters."
+3. AND the related parameter's description:
+    > "Certain versions of this Azure resource reached their end of life. The latest version of this module only includes supported versions of the resource. All unsupported versions have been removed from this parameter."
```

Comment:
I don't think this is necessary.

Reply:
This was also agreed on internally, I just ended up being the one writing it down. :) I think having it in the parameter makes sense, as there's no way not to see it - this was probably the main argument of whoever came up with it.